service/deployer/installer/helm/helm.go

```diff
@@ -219,14 +219,15 @@ func (i *HelmInstaller) Install(event eventerspec.DeploymentEvent) error {
 		return microerror.MaskAny(err)
 	}
 
-	if err := i.runHelmCommand(
-		"install",
-		"install",
-		tarballPath,
+	installCommand := []string{
+		"upgrade",
+		"--install",
 		"--values",
 		valuesFile,
-		"--wait",
-	); err != nil {
+		project,
```

Comment:
Was similar to docker images, like `ferocious-mongoose`. Fun, but needed to go so we could upgrade reliably.